### DISCUSSION TITLE
Displaying State on right side of Template Card

### DISCUSSION THREAD

**Discussion by Arq** [piitaya/lovelace-mushroom#793] [Ideas]
Thank’s for your awesome mushroom template card.
Now in the template card on the left side are rendered two text properties - the Primary and Secondary Information.
However, there is usually plenty of free space on the right side (in horizontal view)
I wonder if it would be possible to render there  additional text property to display e.g. entity.state just like it is on the standard Entities card. As you can see, I want to render as much information as possible about my sensor.
Nowadays:
![Mashroom_template_card-01](https://user-images.githubusercontent.com/102224585/198051080-176ccd26-6c38-4460-afef-721f6bfac0e1.png)
In the future:
![Mashroom_template_card-02](https://user-images.githubusercontent.com/102224585/198050800-22291f11-4e00-4473-9997-cd98d413a074.png)

**Comment by piitaya** [comment]
Mushroom is designed to be used in a grid, not as full width. To keep consistency, the information stay next to the icon. Bottom ans right places are for controls (button, slider, etc...)

**Reply by Arq** [reply]
Ok, I could agree because I'm completely new to HA dashboard design. I'm looking for flexible and fully editable card to display entities with its attributes. Mushroom Template Card is awesome comparing to Entities Card. So maybe next step could be such a **Entities Template Card**.
Interestingly  in my example I didn't use templates defined in HA, The code bellow could be the base class for a list of different battery powered motion sensors, or other sensors if you consider the device_class attribute.
```
type: custom:mushroom-template-card
entity: binary_sensor.your_sensor_id_here
primary: '{{ state_attr(entity,''friendly_name'') }}'
secondary: |-
  {% if states(entity) == 'on' %}
    detected
  {% else %}
    clear
  {% endif %}
  (LQ: {{ state_attr(entity,'linkquality') }})
  {% if state_attr(entity,'battery')!=none %}
    (Bat: {{ state_attr(entity,'battery')}}%)
  {% endif %}
icon: |-
  {% if states(entity) == 'on' %}
    mdi:motion-sensor
  {% else %}
    mdi:motion-sensor-off
  {% endif %}
icon_color: |-
  {% if states(entity) == 'on' %}
    green
  {% else %}
    blue
  {% endif %}
badge_icon: |-
  {% if state_attr(entity,'battery')!=none %}
    {% set baterylevel = state_attr(entity,'battery')  | int  | round(0,'floor') %}
    mdi:battery-{{baterylevel//10}}0
  {% else %}
    {% if state_attr(entity,'battery_low')==true %}
      mdi:battery-low
    {% else %}
      mdi:battery
    {% endif %}
  {% endif %}
badge_color: |-
  {% if state_attr(entity,'battery_low') %}
    red
  {% else %}
    {% if state_attr(entity,'battery')!=none %}
      {% if state_attr(entity,'battery') >60 %}
        green
      {% else %}
        yellow
      {% endif %}
    {% else %}
      blue
    {% endif %}
  {% endif %}
tap_action:
  action: more-info
hold_action:
  action: none
double_tap_action:
  action: none
layout: horizontal
multiline_secondary: false
fill_container: false
```
